app/views/media/_form.html.erb

```diff
@@ -17,18 +16,15 @@
       <%= text_f f, :name %>
       <%= text_f f, :path, :size => "col-md-8", :help_block => _("The path to the medium, can be a URL or a valid NFS server (exclusive of the architecture).
           for example <em>http://mirror.centos.org/centos/$version/os/$arch</em> where <strong>$arch</strong> will be substituted for the host's actual OS architecture and <strong>$version</strong>, <strong>$major</strong> and <strong>$minor</strong> will be substituted for the version of the operating system. Solaris and Debian media may also use <strong>$release</strong>.").html_safe %>
-        <span id="nfs-section" <%= display?(!@medium.operatingsystems.map(&:require_nfs_access_to_medium).any?) %>>
+        <span id="nfs-section" <%= display?(!@medium.operatingsystems.map{ |obj| obj.class.require_nfs_access_to_medium }.any?) %>>
```

Comment:
This won't work if the medium has solaris os_family but no operating systems assigned.
Why not also leverage the `required_nfs_list` helper here?
p.s. you can use `any?` with a block instead of `.map{..}.any?`